docs/iris/src/developers_guide/release.rst

```diff
@@ -0,0 +1,73 @@
+Releases
+********
+
+A release of Iris is a tag on the SciTools/iris Github repository.
+
+Release Branch
+==============
+
+Once the features intended for the release are on master, a release branch should be created, in the SciTools/iris repository.  This will have the name:
+
+    :literal:`{major release number}.{minor release number}.x`
+
+for example:
+
+    :literal:`v1.9.x`
+
+This branch shall be used to finalise the release details in preparation for the release candidate.
+
+Release Candidate
+=================
+
+Prior to a release, a release candidate tag may be created, marked as a pre-release in github, with a tag ending with :literal:`rc` followed by a number, e.g.:
+
+    :literal:`v1.9.0rc1`
+
+If created, the pre-release shall be available for at least one week prior to the release being cut.  User groups should be notified of the existence of the pre-release and encouraged to test the functionality.
+
+A pre-release is expected for a minor release, but not normally provided for a point release.
+
+If new features are required for a release after a release candidate has been cut, a new pre-release shall be issued first.
+
+Documentation
+=============
+
+The documentation should include all of the what's new snippets, which must be compiled into a what's new.  This content should be reviewed and adapted as required and the snippets removed from the branch to produce a coherent what's new page.
+
+Upon release, the documentation shall be added to the SciTools scitools.org.uk github project's gh-pages branch as the latest documentation.
+
+Testing the Conda Recipe
+========================
+
+Before a release is cut, the SciTools conda-recipes-scitools recipe for iris shall be tested to build the release branch of iris; this test recipe shall not be merged onto conda-recipes-scitools.
```

Comment:
@marqh You can choose to ignore me or beat me with a big stick .... but, i believe it should be `Iris` and not `iris`.

If you choose not to ignore me (or beat me with a big stick, thanks) there are other `iris` offenders peppered throughout ...